recipes/jcc/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+export JCC_JDK=$JAVA_HOME
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # OSX recipe
+    #export MACOSX_DEPLOYMENT_TARGET=10.9
+    #export MACOSX_VERSION_MIN=10.9
+
+    export JCC_ARGSEP=";"
+    export JCC_INCLUDES="$PREFIX/include;$PREFIX/include/darwin"
+	export JCC_LFLAGS="-v;-L$PREFIX/jre/lib;-ljava;-L$PREFIX/jre/lib/server;-ljvm;-Wl,-rpath;-Wl,$PREFIX/jre/lib;-Wl,-rpath;-Wl,$PREFIX/jre/lib/server;-mmacosx-version-min=$MACOSX_DEPLOYMENT_TARGET"
+	export JCC_CFLAGS="-fno-strict-aliasing;-Wno-write-strings;-Qunused-arguments;-mmacosx-version-min=$MACOSX_DEPLOYMENT_TARGET"
+    export NO_SHARED=1
+    printenv
+
+    $PYTHON setup.py install --single-version-externally-managed --record record.txt
+
+else
+    # GNU/Linux recipe
+    export JCC_ARGSEP=";"
+	export JCC_INCLUDES="$PREFIX/include;$PREFIX/include/linux"
```

Comment:
Aren't these set in setup.py?

Reply:
Yes, setup would cover these, but they are "hardcoded" from JCC_JDK in setup.py so I thought it was better to be explicit? Can remove if preferred.

Reply:
removed!

Reply:
the export JCC_ARGSEP stuff seems to be needed.